app/src/layout/PublicUser.tsx

```diff
@@ -103,6 +105,13 @@ const PublicUserComponent: React.FC<PublicUserComponentProps> = ({
       { enabled: !!enableReports, staleTime: Infinity },
     );
 
+  const { data: marriages } = api.marriage.getMarriedUsers.useQuery(
+    { id: userId },
+    {
+      staleTime: 5000,
```

Comment:
Again I think we could bump this to 5 minutes :)

Reply:
Bumped all timers to 5 minutes =)